<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-GC5HGKU" failures="1" tests="3" name="Test" time="16.736" errors="0" timestamp="2023-06-07T09:16:03 IST">
  <testcase classname="RahulShetty.ExtendReport.test2" name="Initialdemo" time="15.816">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at RahulShetty.ExtendReport.test2.Initialdemo(test2.java:29)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:833)
... Removed 16 stack frames]]>
    </failure>
  </testcase> <!-- Initialdemo -->
  <testcase classname="RahulShetty.ExtendReport.test1" name="Initialdemo" time="16.568"/>
  <testcase classname="RahulShetty.ExtendReport.test3" name="Initialdemo" time="16.664"/>
</testsuite> <!-- Test -->
